test/e2e/tests/tokens/custom-token-send-transfer.spec.ts

```diff
@@ -0,0 +1,382 @@
+          const sendTokenPage = new SendTokenPage(driver);
+          const confirmTxPage = new ConfirmTxPage(driver);
+          const activityListPage = new ActivityListPage(driver);
+
+          await homePage.check_pageIsLoaded();
+
+          await assetListPage.openTokenDetails(symbol);
+          await assetListPage.clickSendButton();
+
+          await sendTokenPage.check_pageIsLoaded();
+          await sendTokenPage.fillRecipient(recipientAddress);
+          await sendTokenPage.fillAmount('1');
+          await sendTokenPage.clickContinueButton();
+
+          // check transaction details
+          const estimatedGasFee = '0.00008455';
+          const totalAmount = `${valueWithSymbol('1')} + 0.00008455`;
+          await confirmTxPage.check_pageIsLoaded(estimatedGasFee, totalAmount);
+
+          // check function name and hex data details in hex tab
+          await confirmTxPage.check_functionTypeAndHexData(
+            'Transfer',
+            '0xa9059cbb0000000000000000000000002f318c334780961fb129d2a6c30d0763d9a5c97',
+          );
+
+          // edit gas fee
+          await confirmTxPage.switchToDetailsTab();
+          await confirmTxPage.editGasFee('60000', '10');
+          await confirmTxPage.confirmTx();
+
+          // check that transaction has completed correctly and is displayed in the activity list
+          await activityListPage.check_txAction(`Send ${symbol}`);
+          await activityListPage.check_txAmountInActivity(
+            valueWithSymbol('-1'),
+          );
+        },
+      );
+    });
+
+    it('transfer custom tokens from dapp customizing gas values', async function () {
+      await withFixtures(
+        {
+          dapp: true,
+          fixtures: new FixtureBuilder()
+            .withPermissionControllerConnectedToTestDapp()
+            .withTokensControllerERC20()
+            .build(),
+          ganacheOptions: defaultGanacheOptions,
+          smartContract,
+          title: this.test?.fullTitle(),
+        },
+        async ({ driver, contractRegistry }) => {
+          const contractAddress = await contractRegistry.getContractAddress(
+            smartContract,
+          );
+          await unlockWallet(driver);
+
+          await tempToggleSettingRedesignedTransactionConfirmations(driver);
+
+          const testDapp = new TestDapp(driver);
+          const confirmTxPage = new ConfirmTxPage(driver);
+          const activityListPage = new ActivityListPage(driver);
+          const homepage = new HomePage(driver);
+          const assetListPage = new AssetListPage(driver);
+
+          // transfer token from dapp
+          await openDapp(driver, contractAddress);
+          await testDapp.check_pageIsLoaded();
+          await testDapp.clickTransferTokens();
+
+          await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+
+          const estimatedGasFee = '0.00010321';
+          const totalAmount = `${valueWithSymbol('1.5')} + 0.00010321`;
```

Comment:
what about using estimateGasFee variable on line 118, so it's not hardcoded in different places, wdyt?